src/music21/meter.ts

```diff
@@ -250,9 +256,11 @@ export class TimeSignature extends base.Music21Object {
                     beamType = 'partial-right';
                 }
             } else if (isLast) {
-                beamType = 'start';
+                beamType = 'stop';
                 if (beamPrevious === undefined || !beamPrevious.getNumbers().includes(beamNumber)) {
                     beamType = 'partial-left';
+                } else if (beamPrevious && beamPrevious.getTypeByNumber(beamNumber) === 'stop') {
+                    beamsList[i] = undefined;
```

Comment:
When the final beat of the final measure is incomplete (common with pickup measures), this check ensures that the last note is not beamed. (E.g., three 8th notes in 2/4)

Reply:
Thank you!